setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup
+
+setup(
+    name='govstat',
+    version='0.0.1',
+    author='stevesdawg',
+    long_description='',
+    description='',
+    zip_safe=False,
+    packages=['app'],
+    install_requires=[
+        "flask",
+        "flask_sqlalchemy",
+        "flask_migrate",
+        "requests",
+        "pandas",
+        "numpy",
```

Comment:
I would also add the following packages:
- xlrd
- flask-wtf
- pymysql

Reply:
got it